pkg/controller/installation/pod_cidr_detection.go

```diff
@@ -0,0 +1,44 @@
+package installation
+
+import (
+	"net"
+	"regexp"
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+const (
+	// KubeadmConfigConfigMap is defined in k8s.io/kubernetes, which we can't import due to versioning issues.
+	kubeadmConfigMap = "kubeadm-config"
+)
+
+// extractKubeadmCIDRs looks through the config map and parses lines starting with 'podSubnet'.
+func extractKubeadmCIDRs(kubeadmConfig *v1.ConfigMap) ([]string, error) {
+	var line []string
+	var foundCIDRs []string
+
+	// Look through the config map for lines starting with 'podSubnet', then assign the right variable
```

Comment:
nit: `lines` should be `a line`
With the comment saying `lines` I thought there could be multiple.